src/application/api_0_1_0/routes.py

```diff
@@ -27,6 +27,23 @@ def post(self):
         new_user = service.create_user(data)
         return {'tokenId': new_user.uuid}, 201
 
+class UserResource(Resource):
+    """
+    Resource to get a user.
+
+    The API should return user details
+    (first name, last name, email)
+    """
+
+    def get(self, uuid):
+        service = UserService()
+        user = service.filter_by_uuid(uuid)
+
+        print user
```

Comment:
please make sure you don't have any `print()` statements or debuggers in your code.